### DISCUSSION TITLE
Template with authentication ?

### DISCUSSION THREAD

**Discussion by Nizar-Almshantaf** [ardalis/CleanArchitecture#675] [Q&A]
does anyone have this template with authentication ready (in any approach or using identityserver)?

**Comment by ardalis** [comment]
We had a PR that added it but it didn't end up getting merged and the template evolved a lot so it would be hard to add now. I'm interested in adding it myself but just haven't gotten to it.

Does anyone else have something they can share?

You can look at how auth is done in the eShopOnWeb sample, which also uses Clean Architecture. It's not a lot of code to add.
